jobs/utils/snowflake.py

```diff
@@ -197,20 +207,34 @@ def gdf_to_snowflake(
         for c, dtype in gdf.dtypes.to_dict().items():
             if type(dtype) == geopandas.array.GeometryDtype:
                 if epsg == WGS84:
-                    cols.append(f'TO_GEOGRAPHY("{c}") AS "{c}"')
+                    cols.append(
+                        f'{"" if strict_geometries else "TRY_"}TO_GEOGRAPHY("{c}") AS "{c}"'
```

Comment:
There were some invalid building footprints in the dataset, no matter how hard I tried to make them valid. So I think it makes sense, in the spirit of ELT, to go ahead and allow the invalid ones through, cast them using [`TRY_TO_GEOGRAPHY`](https://docs.snowflake.com/en/sql-reference/functions/try_to_geography), and filtering out nulls later.

Reply:
That makes sense to me, thank you for trying and for explaining your final decision.